### PR TITLE
refac: rm 2930 and 7702 - use alloy-rs/eips

### PR DESCRIPTION
**Pull request by yash-atreya**
## Motivation

Ref #1174. 

## Solution

2930 and 7702 have been extracted to a new repo - https://github.com/alloy-rs/eips. Use that and re-export in alloy-eips.

## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation
- [ ] Breaking changes

### CODE REVIEW

File: crates/alloy/Cargo.toml [hunk 1]
```diff
@@ -269,7 +269,6 @@ arbitrary = [
 k256 = [
     "alloy-core/k256",
     "alloy-consensus?/k256",
-    "alloy-eips?/k256",
```
Comment (mattsse) [inline crates/alloy/Cargo.toml:272]: where did this go?

File: crates/eips/Cargo.toml [hunk 2]
```diff
@@ -19,6 +19,10 @@ rustdoc-args = ["--cfg", "docsrs"]
 workspace = true
 
 [dependencies]
+# eips
+eip-2930 = { workspace = true }
+eip-7702 = { workspace = true }
```
Comment (mattsse) [inline crates/eips/Cargo.toml:24]: .workspace

File: crates/eips/Cargo.toml
Comment (mattsse) [inline crates/eips/Cargo.toml:1]: please avoid reformatting entries for these prs, makes it much harder to review

**Review by mattsse — CHANGES_REQUESTED** [review]

File: crates/alloy/Cargo.toml (hunk 1 above)
Comment (yash-atreya) [reply crates/alloy/Cargo.toml:272]: Oversight, reintroduced

**Review by onbjerg — COMMENTED** [review]
let's update the alloy eips readme to point to the new repo so people know where to look for the code

**Review by onbjerg — APPROVED** [review]

**Comment by yash-atreya** [comment]
Bumping `alloy-rs/eips` further requires #1167

**Review by mattsse — APPROVED** [review]

**Pull request merged**